src/utils/ApiEndpoints.js

```diff
@@ -1,18 +1,18 @@
 const BASE_URL = "https://hacker-news.firebaseio.com";
 
 class ApiEndpoints {
-    static getNewsItem(itemId) {
+    static getNewsItem(itemId, callback) {
         let request = new Request(`${BASE_URL}/v0/item/${itemId}.json`);
         fetch(request)
             .then(response => {
                 if (response.status === 200) {
-                    return response.json();
+                    return response.json();;
```

Comment:
Nitpicking: please remove the second semicolon.

Reply:
done, thank you